csharp-intermediate-topics/UnsafeCodeInCSharp/Tests/Tests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnsafeCodeInCSharp.Tests
+{
+    [TestClass]
+    public class Tests
+    {
+        [TestMethod]
+        public void WhenPointerVariablePassedThenValueTriples()
```

Comment:
Can this follow the normal test name pattern of Given_When_Then please Satya? The Given part is optional, so simply a matter of separating the When and Then parts with `_`.